Makefile

```diff
@@ -3,6 +3,10 @@
 VALIDATOR_IMAGE := "ghcr.io/peggyjv/sommelier-sommelier:main"
 ORCHESTRATOR_IMAGE := "ghcr.io/peggyjv/gravity-bridge-orchestrator:main"
 
+build_protos:
+	chmod u+x ./build_protos.sh
```

Comment:
If you check this file in with `+x` I believe it will be stored that way, we shouldn't need to `chmod` every time someone runs `make`.